cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/Cargo.toml

```diff
@@ -42,6 +42,7 @@ emulated-integration-tests-common = { path = "../../../common", default-features
 rococo-westend-system-emulated-network = { path = "../../../networks/rococo-westend-system" }
 rococo-system-emulated-network = { path = "../../../networks/rococo-system" }
 asset-hub-rococo-runtime = { path = "../../../../../runtimes/assets/asset-hub-rococo", default-features = false }
+penpal-emulated-chain = { path = "../../../chains/parachains/testing/penpal", default-features = false }
```

Comment:
@claravanstaden I think no need to add this deps here, because it can be reached from ` rococo-system-emulated-network`, I will try it in my branch